src/components/NavigationMenu/NavigationMenu.scss

```diff
@@ -1,9 +1,9 @@
-.navigation-menu ul {
+.navigation-menu__links {
   display: flex;
   gap: 15px;
 }
 
-.link {
+.navigation-menu__link {
```

Comment:
¿Para qué estáis usando SASS?

Reply:
Si tan solo tuvieras una manera de comprobarlo...

Reply:
![ApprovedCheckGIF](https://github.com/MarcosRodriguezAlonso/the-why-blog/assets/128187875/7ecaac99-02f1-41df-9680-741bb10bf2e3)